bcipy/parameters/lm_params.json

```diff
@@ -0,0 +1,67 @@
+{
+  "kenlm_model": {
+    "value": "lm_dec19_char_large_12gram.kenlm",
+    "section": "lm_config",
+    "readableName": "KenLM Model File",
+    "helpTip": "Name of the pretrained model file",
+    "recommended_values": [
+      "lm_dec19_char_large_12gram.kenlm"
+    ],
+    "type": "filepath"
+  },
+  "causal_model_name": {
+    "value": "facebook/opt-125m",
+    "section": "lm_config",
+    "readableName": "Causal Model Name",
+    "helpTip": "Name of the HuggingFace model to load. Required for the Causal model. Default: facebook/opt-125m",
+    "recommended_values": [
+      "facebook/opt-125m",
+      "gpt2",
+      "distilgpt2"
+    ],
+    "type": "str"
+  },
+  "causal_model_path": {
+    "value": "",
+    "section": "lm_config",
+    "readableName": "Language Model Name",
+    "helpTip": "Pretrained model path, relative to causal.py directory. Leave blank if using HuggingFace model",
+    "recommended_values": [
+      "out_subtitle_distil_packed_batch4_fixed_pack_prob0.80/"
+    ],
+    "type": "directorypath"
+  },  
+  "mixture_types": {
```

Comment:
These could live closer together in the mixture params since they are directly related, especially if we already support a list of dictionaries. 

```
{
"mixture": 
 "recommended_values": [
      "{}, {}",
      "{'lang_model_name': 'gpt2', 'weight': 0.1, 'type': 'CAUSAL'}, "{ 'weight': 0.9, 'type': 'KENLM'},"
}
```